main.js

```diff
@@ -0,0 +1,24 @@
+// let els = document.querySelectorAll("a");
+// for(let el of els){
+//     if(el.textContent.includes("All Posts")){
+//         el.remove();
+//     }
+// }
+
+let el = document.querySelector("header nav ul li");
+el.remove();
+
+let els = document.querySelectorAll("span");
+for(let el of els){
+    if(el.textContent.includes("Read Time:")){
+        el.remove();
+    }
+}
```

Comment:
```suggestion
let els = document.querySelectorAll(".featured span");
els.remove()
```
The readme specifies that you should only remove the "featured" Read Time `<span>`.